build-aux/com.usebottles.bottles.json

```diff
@@ -247,6 +159,7 @@
         },
         {
             "name": "ImageMagick",
+            "buildsystem": "autotools",
```

Comment:
`autotools` is implicitly used, so we don't need to explicitly set it

Reply:
it's just to be more clear, i couldn't find in the flatpak builder docs about being the default for buildsystem
also the module right above is explicitly set as autotools